defaults.yml

```
@@ -1053,4 +1073,4 @@ categories:
               width: 696
               height: 252
             text: Repeats the key as long as it is held down.
-        versions: [Monterey, Big Sur]
+        versions: [Monterey, Big Sur]z
```

```suggestion
        versions: [Monterey, Big Sur]
```
Oops, it must be a mistake.

---

Sounds good. I am guessing it will work on Big Sur as well. I have not tested.

---

Actually you forgot a `z` there 😄